diff --git a/node_modules/react-native-material-buttons/src/components/button/index.js b/node_modules/react-native-material-buttons/src/components/button/index.js
index 40d89e6..0dcd062 100644
--- a/node_modules/react-native-material-buttons/src/components/button/index.js
+++ b/node_modules/react-native-material-buttons/src/components/button/index.js
@@ -69,6 +69,7 @@ export default class Button extends PureComponent {
         toValue: 1,
         duration: focusAnimationDuration,
         easing: Easing.out(Easing.ease),
+        useNativeDriver: false
       })
       .start();
   }
@@ -82,6 +83,7 @@ export default class Button extends PureComponent {
         toValue: 0,
         duration: focusAnimationDuration,
         easing: Easing.out(Easing.ease),
+        useNativeDriver: false
       })
       .start();
   }
@@ -104,7 +106,7 @@ export default class Button extends PureComponent {
     });
 
     let rippleStyle = {
-      backgroundColor: props.disabled? disabledColor : color,
+      backgroundColor: props.disabled ? disabledColor : color,
     };
 
     let shadeStyle = {
@@ -123,7 +125,7 @@ export default class Button extends PureComponent {
         onPressOut={this.onPressOut}
       >
         {children}
-        <Animated.View style={[ styles.shade, shadeStyle ]} />
+        <Animated.View style={[styles.shade, shadeStyle]} />
       </Ripple>
     );
   }
